src/libraries/System.Net.NameResolution/src/ExcludeApiList.PNSE.Browser.txt

```diff
@@ -1 +1 @@
-M:System.Net.Dns.GetHostName
+System.Net.Dns.GetHostName
```

Comment:
Why aren't we using the same format as before? This format seems to be a pretty standard format - we use it in `SuppressMessageAttribute` for code analysis - and the ILLinker adopted it as well.

[https://github.com/dotnet/csharplang/blob/master/spec/documentation-comments.md#id-string-format](https://github.com/dotnet/csharplang/blob/master/spec/documentation-comments.md#id-string-format)

Reply:
Yeah I like this format as well... even if we just remove the prefix in the generator.

Reply:
Roslyn visitor pattern doesn't provide a good way to get these docIds.
We will have to manually implement that as well but i think its bit too much to just support this.
Maybe we can ask roslyn to provide a good way to get the fully qualified name(or maybe i need more searching). Currently i have to just traverse the graph to get the name.

Reply:
> we just remove the prefix in the generator.

we can implement that.

Reply:
For that you would need to use the semantic model which I don't think we want this:

https://github.com/safern/ApplyNullableDecorators/blob/master/src/NullableDecoratorVisitor.cs#L37-L38

Reply:
Here's the code in the ILLinker that does this (it was copied from roslyn):

https://github.com/mono/linker/blob/95d8c21c8d9bfc8cad107ff36fa44851f61bb28c/src/linker/Linker/DocumentationSignatureGenerator.cs

Reply:
I will try to add this for 6.0, i wont be able to get this in for rc1 & we will require this feature for rc1.
I can add the filtering to support docids, so in future we will just have to correct the arcade version

Reply:
Sounds good to me.